app/models/job.rb

```diff
@@ -1,4 +1,46 @@
 class Job < ApplicationRecord
   validates :title, :level_of_interest, :city, presence: true
   belongs_to :company
+  belongs_to :category
+  has_many :comments
+
+  def sorted_comments
+    comments.order(:created_at).reverse
+  end
+
+  def self.sort_by(filter)
+    return order(:city) if filter == "location"
+    order(level_of_interest: :desc) if filter == "interest"
```

Comment:
These guard statements can sometimes get a little bit out of control. Since you're testing both of these for equality against `filter` I might reach for a case statement instead. Assuming we add filters in the future, a case statement seems to more closely reflect what we're trying to do in this method.